Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -691,6 +694,134 @@ public class CameraManager {
+        let propertyAnimator = UIViewPropertyAnimator(duration: duration, dampingRatio: ratio, animations: animations)
+        let cameraAnimator = CameraAnimator(delegate: self, propertyAnimator: propertyAnimator, owner: animationOwner)
+        cameraAnimators.add(cameraAnimator)
+        return cameraAnimator
+    }
+
+    /**
+     Convenience to create a `CameraAnimator` and will add it to a List of `CameraAnimators` to track the lifecycle of that animation
+
+     - Parameter duration: The duration of the animation, in seconds.
+     - Parameter delay: The number of seconds to wait before starting the animations. Specify 0 to begin the animations immediately.
+     - Parameter options: The options to apply to the animations. You can specify most options, but transition-related options and options related to the animation direction are ignored.
+     - Parameter animationOwner: The `id` of what owns this animation.
+     - Parameter animations: The block containing the animations. This block has no return value and takes no parameters. Use this block to modify any animatable view properties. When you start the animations, those properties are animated from their current values to the new values using the specified animation parameters.
+     - Parameter completion: Completion block to be passed through for when an animation is stopped
+     - Returns `CameraAnimator`: A class that represents an animator with the provided configuration.
+     */
+    public func runningCameraAnimator(duration: TimeInterval,
+                               delay: TimeInterval,
+                               options: UIView.AnimationOptions = [],
+                               animationOwner: AnimationOwnerProtocol = AnimationOwner.unspecified,
+                               animations: @escaping () -> Void,
+                               completion: AnimationCompletion? = nil) -> CameraAnimator {
+        let runningAnimator = UIViewPropertyAnimator.runningPropertyAnimator(withDuration: duration,
+                                                                             delay: delay,
+                                                                             options: options,
+                                                                             animations: animations,
+                                                                             completion: nil)
+
+        let cameraAnimator = CameraAnimator(delegate: self, propertyAnimator: runningAnimator, owner: animationOwner)
+
+        runningAnimator.addCompletion({ [weak self] animatingPosition in
+            guard let completion = completion,
+                  let self = self
+            else { return }
+
+            self.schedulePendingCompletion(forAnimator: cameraAnimator, completion: completion, animatingPosition: animatingPosition)
+        })
+        cameraAnimators.add(cameraAnimator)
+        return cameraAnimator
+    }
+
+    // MARK: Delegate Functions
```

Comment:
please provide more detail about which delegate functions

Reply:
Could we reword the "MARK: Delegate functions" to "CameraAnimatorDelegate functions"

Reply:
yeah my bad, forgot this one